### PR TITLE
[BUGFIX] Change controllerActionName in extbase configuration of gallery

### LINKED ISSUE #887: Gallery not compatible with TYPO3 v10
# Bug Report

## Prerequisites

* [ ] Can you reproduce the problem on TYPO3 v8.7 LTS
* [ ] Can you reproduce the problem on TYPO3 v9.5 LTS
* [x] Can you reproduce the problem on TYPO3 v10.4 LTS
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Abenjaminkott+repo%3Abenjaminkott%2Fbootstrap_package)
  to see if your bug or enhancement is already reported?

## Description

The content element 'Gallery' does not work anymore after upgrading TYPO3 to v10.4 LTS.

## Steps to Reproduce

1. Add a content element of type 'Gallery'
2. Open page in frontend

### Actual behavior

Error message:
```
(1/1) #1540246570 TYPO3Fluid\Fluid\Core\ViewHelper\Exception
No Content Object definition found at TypoScript object path "tt_content.gallery.20"
```

## Versions

TYPO3 v10.4.2
bootstrap_package: master#6cded41

### PR DESCRIPTION
**Pull request by dahaupt**
Fixes #887

### CODE REVIEW

File: ext_localconf.php [hunk 1]
```diff
@@ -264,8 +264,8 @@
  */
 \TYPO3\CMS\Extbase\Utility\ExtensionUtility::configurePlugin(
     'BK2K.BootstrapPackage',
-    'Content',
+    'Gallery',
     [
-        'Content' => 'generic',
+        'Gallery' => 'gallery',
```
[the remark this answers is not in the document]
Comment (alexanderschnitzler) [reply ext_localconf.php:269]: True, this change is wrong. Only the action name has to be changed, the controller is fine.

File: Configuration/TypoScript/ContentElement/Element/Gallery.typoscript [hunk 2]
```diff
@@ -14,11 +14,13 @@ tt_content.gallery {
...
     ###############
+    # Currently, the pagination does not work without an extbase environment.
+    # For details see: https://forge.typo3.org/issues/79430
     extbase {
         controllerExtensionName = BootstrapPackage
-        controllerName = Content
-        controllerActionName = generic
-        pluginName = Content
+        controllerName = Gallery
+        controllerActionName = gallery
```
[the remark this answers is not in the document]
Comment (alexanderschnitzler) [reply Configuration/TypoScript/ContentElement/Element/Gallery.typoscript:22]: You can leave the controller name as is and just adjust the action name.

**Comment by dahaupt** [comment]
@gilbertsoft Looking forward to your solution with the native paginator!

@alexanderschnitzler The extbase action is never executed and only the template of the action is rendered. Correct me if I am worng, but even when deleting the controller class everything works fine.

**Comment by alexanderschnitzler** [comment]
@dahaupt Yes, the action is not really called and not necessary, at least in version 10. Didn't check the behavior of version 9.5.

**Comment by gilbertsoft** [comment]
> 
> 
> @dahaupt Yes, the action is not really called and not necessary, at least in version 10. Didn't check the behavior of version 9.5.

According to @benjaminkott this is a workaround for 9.5

**Pull request merged**